velox/docs/monitoring/stats.rst

```diff
@@ -0,0 +1,34 @@
+
+=============
+Runtime Stats
+=============
+
+Runtime stats are used to collect the per-query velox runtime events for
+offline query analysis purpose. The collected stats can provide insights into
+the operator level query execution internals, such as how much time a query
+operator spent in disk spilling. The collected stats are organized in a
+free-form key-value for easy extension. The key is the event name and the
+value is defined as RuntimeCounter which is used to store and aggregate a
+particular event occurrences during the operator execution. RuntimeCounter has
+three types: kNone used to record event count, kNanos used to record event time
+in nanoseconds and kBytes used to record memory or storage size in bytes. It
+records the count of events, and the min/max/sum of the event values. The stats
+are stored in OperatorStats structure. The query system can aggregate the
+collected stats based on stats key and operator type for gianalysis.
+
+Memory Arbitration
```

Comment:
Perhaps, clarify that these stats are available for all operators. There are operator-specific stats that we'll add now that you have started the docs.